### PR TITLE
small change to test

### PR DESCRIPTION
**Pull request by korinan**

### CODE REVIEW

**Comment by oleksandr-korin** [comment]
adfsasfd

File: index.html [hunk 1]
```diff
@@ -62,7 +62,8 @@ <h3 class="sidebar-title">Soft Skills</h3>
 
         </aside>
 
-        <!--Main content section-->
```
Comment (oleksandr-korin) [inline index.html:65]: good code

Comment (korinan) [reply index.html:65]: dsadasd

Comment (korinan) [reply index.html:65]: zxczxc

**Pull request merged**